doit.go

```diff
@@ -40,26 +44,115 @@ func (c *LiveConfig) GetGodoClient() *godo.Client {
 	return godo.NewClient(oauthClient)
 }
 
+func sshConnect(user string, host string, method ssh.AuthMethod) (err error) {
+	sshc := &ssh.ClientConfig{
+		User: user,
+		Auth: []ssh.AuthMethod{method},
+	}
+	conn, err := ssh.Dial("tcp", host, sshc)
+	if err != nil {
+		return err
+	}
+
+	session, err := conn.NewSession()
+	if err != nil {
+		return err
+	}
+
+	session.Stdout = os.Stdout
+	session.Stderr = os.Stderr
+	session.Stdin = os.Stdin
+
+	defer session.Close()
+
+	modes := ssh.TerminalModes{
+		ssh.ECHO: 1,
+	}
+
+	var (
+		termWidth, termHeight int
+	)
+	fd := os.Stdin.Fd()
+	if term.IsTerminal(fd) {
+		oldState, err := term.MakeRaw(fd)
+		if err != nil {
+			return err
+		}
+
+		defer term.RestoreTerminal(fd, oldState)
+
+		winsize, err := term.GetWinsize(fd)
+		if err != nil {
+			termWidth = 80
+			termHeight = 24
+		} else {
+			termWidth = int(winsize.Width)
+			termHeight = int(winsize.Height)
+		}
+	}
+
+	if err := session.RequestPty("xterm", termWidth, termHeight, modes); err != nil {
+		session.Close()
+		return err
+	}
+	if err == nil {
+		err = session.Shell()
+	}
+	if err != nil {
+		return err
+	}
+
+	err = session.Wait()
+	if err != nil && err != io.EOF {
```

Comment:
you need to handle `io.EOF` differently, else you will be returning it instead of `nil`